tests/serverpod_test_server/test_e2e/database_roundtrip_operations_test.dart

```diff
@@ -0,0 +1,115 @@
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:test/test.dart';
+
+Future<void> _setupTestData(Client client) async {
+  await client.basicDatabase.deleteAllSimpleTestData();
+  await client.basicDatabase.createSimpleTestData(100);
+}
+
+void main() {
```

Comment:
Suggestion: Could you validate that we have this type of check in an integration test. I think an integration test is better and good enough for database operations.

Previously we did not have database query tests outside of end-to-end tests. But it is also hard to get full coverage of all different database queries when doing it end-to-end. Hence we have moved into putting this into integration tests, skipping the serialization in endpoints.

Reply:
I did not find a similar integration test to all the cases we did with `Types` here, so extended `tests/serverpod_test_server/test_integration/database_operations/crud/read_test.dart` in that regard.